lib/base_service.js

```diff
@@ -276,9 +276,11 @@ BaseService.prototype._requireModule = function(modName) {
     try {
         return P.resolve(require(modName));
     } catch (e) {
-        if (/^\//.test(opts.mod) || (opts.baseTried && opts.modsTried)) {
-            // we have a full path here which can't be required, or we tried
-            // all of the possible combinations, so bail out
+        if (/^\//.test(opts.mod) || (opts.baseTried && opts.modsTried) ||
+                e.message !== "Cannot find module '" + modName + "'") {
```

Comment:
Hm, right, nested dependencies will produce the same error code, didn't think about that..